index.bs

```diff
@@ -31,7 +31,9 @@ This document contains a set of questions
 intended to help <abbr title="specification">spec</abbr> authors
 as they think through
 the security and privacy implications
-of their work and write the narrative Security Considerations and Privacy Considerations sections for inclusion in-line in their specifications, as described below in [[#considerations]].
+of their work and write the narrative Security Considerations and Privacy
```

Comment:
Would like @alice's thoughts on this line breaking.

Reply:
Ah, tbh i didn't pay any mind at all to line breaking, i just kept wrapping to 80 cols.  If there is a method, pattern for it, i'm sure i've goofed it up.  If there is guidance somewhere on correct link breaking in bikeshed, i'd be grateful for any pointers / links :)